packages/ag-charts-community/src/chart/overlay/overlay.ts

```diff
@@ -18,12 +18,21 @@ export class Overlay {
     @Validate(STRING, { optional: true })
     text?: string;
 
+    @Validate(BOOLEAN)
+    darkMode = false;
+
     show(rect: BBox) {
         if (!this.element) {
             this.element = this.createElement('div');
             this.element.className = this.className;
         }
 
+        if (this.darkMode) {
```

Comment:
for these if else statements maybe we could use this one line instead? `this.element.classList.toggle(`${this.className}-dark`, this.darkMode);`

Reply:
Good call, done